Add Fractal Tree versions

villares opened the pull request:
I'm very unsure about header style and I will be glad to make any corrections.

And please also forgive any git repo etiquette *faux pas*, I still struggle with simple things like updating my fork to upstream.

ref: issue #50

villares committed:
Merge pull request #1 from jeremydouglass/master

get updates from Jeremy

villares committed:
adding Python mode examples

Thanks for having us Jeremy!

villares committed:
Delete sketch.properties

villares committed:
Revert "adding Python mode examples"

This reverts commit 8ed9d63adbd1afe0f0f2244d79de50374166d746.

villares committed:
Merge remote-tracking branch 'upstream/master'

villares committed:
adding FractalTree variations

jeremydouglass commented:
Great -- will review soon!

> repo etiquette _faux pas_

No worries -- I am still learning myself. My main goal is that your commits always end up in the tree with you as the committer, crediting your work.

Your one major pull request #47 with the reorganization I didn't merge -- I cherry picked your commits and then closed the pull -- but from here on we'll try to make sure that your pulls themselves are worked on and merged so that they will show as merged in GitHub.

villares committed:
renaming ro meet convention and header style changes

Capitalization of filename after underscore
Task title and sub-title separated by " -- "

villares committed:
to force case rename!

villares committed:
now the correct filename case :/

jeremydouglass commented:
Okay, wow, I see what was going on here. I rebased several times and it kept generating really weird merge conflict sets. git really doesn't like file renaming that is case-change-only.

In the future, the way to deal with that is

> git mv -f MyFile_path/MyFile_name MyFile_Path/MyFile_Name

Anyway: Closed by e369f5dc5 , db6181741 , 6fa051851
